views/js/OystOneClick.js

```diff
@@ -62,7 +61,7 @@ function OystOneClick(url, productId) {
             available = false;
         }
 
-        if (productExported.isExported && available) 
+        if (available) 
```

Comment:
Why removing the condition with isExported ? This one is set on the server side to know if the product has been sent to oyst..